cmd/node/main.go

```diff
@@ -1356,6 +1356,8 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		processComponents.TxLogsProcessor.EnableLogToBeSavedInCache()
 	}
 
+	maxTxValueStringLength := len(economicsConfig.GlobalSettings.GenesisTotalSupply)
```

Comment:
Actually the theoretically max supply will be bigger than genesis supply but actually it will have the same number of digits (31 mil vs 20 mil)

Reply:
I know, I thought of this and the number of characters is the same. Maybe I can increase it with `1` in case we want to start a testnet with a total supply of let's say `999` and after a few epochs there will be a different number of characters